src/filters/byte-format.js

```diff
@@ -0,0 +1,13 @@
+import { module } from 'angular';
+
+export default module('app.filters.byte-format', [])
+    .filter('byteFormat', () => (bytes, precision) => {
+      let myPrecision = precision;
+      const units = ['bytes', 'kB', 'MB', 'GB', 'TB', 'PB'];
+      const number = Math.floor(Math.log(bytes) / Math.log(1024));
+      if (isNaN(parseFloat(bytes)) || !isFinite(bytes)) return '-';
+      if (bytes === 0) return '0 MB';
```

Comment:
let's move this to the top and add a `if (bytes === undefined) return undefined;` (this may happen when the data is not yet ready for a template)